src/components/atoms/MessageText/MessageText.tsx

```diff
@@ -0,0 +1,36 @@
+import { Typography, Box } from "@mui/material";
+import styles from "./MessageText.module.scss";
+
+type MessageTextProps = {
+  content: string;
+  date: Date;
+  isLast?: boolean;
+};
+
+export const MessageText = ({ content, date, isLast }: MessageTextProps) => {
+  return (
+    <Box className={styles.messageText}>
+      <Typography variant="body1" noWrap component="div">
+        {content}
+      </Typography>
+      {isLast ? <p className={styles.date}>{dateAsString(date)}</p> : <></>}
+    </Box>
+  );
+};
+
+const dateAsString = (inputDate: Date): string => {
+  const [day, month, year, hour, minutes] = [
+    inputDate.getDate(),
+    inputDate.getMonth() + 1,
+    inputDate.getFullYear(),
+    inputDate.getHours(),
+    inputDate.getMinutes(),
+  ];
+  const date = `${day <= 9 ? "0" + day : day}.${
+    month <= 9 ? "0" + month : month
+  }.${year}`;
+  const time = `${hour <= 9 ? "0" + hour : hour}:${
+    minutes <= 9 ? "0" + minutes : minutes
+  }`;
```

Comment:
co to robi? :)

Reply:
wyciąga datę z i czas z obiektu i zapisuje w formacie górne: 
DD.MM.YYYY
a dolne
HH:MM
Później jest to przedstawiane na ekranie jako:
DD.MM.YYYY HH:MM
Nie mogłem nigdzie znaleźć odpowiedniej funkcji do wyświetlania tego w takim formacie xD

Reply:
Myślę, że może być, ale w takiej sytuacji nadaje się do wyeksportowania do osobnego pliku (np w tym komponencie albo folderu `src/utils` i napisanie do tego testu :)

Warto też spróbować wykorzystać coś z tego. Zwykle dobrze się sprawdza :) https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_tolocalestring_date_all

Reply:
Jedyne co to zmieniłbym nazwę funkcji (coś w stylu daty w wiadomości), ale nic nie przychodzi mi teraz  do głowy. :)